src/repository/base.repository.js

```diff
@@ -72,8 +94,8 @@ class BaseRepository {
    * @returns {object} - returns a database object
    * @memberof BaseRepository
    */
-  static async findAll(model, options) {
```

Comment:
Good Job so far @tobechukwuobitube . However, I noticed the baseRepository findAll method was removed, I think you need to confirm no one else used it before it get merged.